dockerfiles/run/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
       - rabbitmq
       - swift
```

Comment:
Did you try to run this locally? I doubt it works with this old swift object storage version